judgels-backends/uriel/uriel-app/src/main/java/judgels/uriel/contest/clarification/ContestClarificationStore.java

```diff
@@ -30,6 +30,12 @@ public ContestClarification createClarification(String contestJid, ContestClarif
                 ContestClarificationStore::fromModel);
     }
 
+    public List<ContestClarification> getAllClarifications(String contestJid) {
```

Comment:
Consistency: `getClarifications()`; 'all' is already implied by the overload signature